compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/platform/WebTextInputService.kt

```diff
@@ -18,21 +18,21 @@ package androidx.compose.ui.platform
 
 import androidx.compose.ui.geometry.Offset
 import androidx.compose.ui.geometry.Rect
+import androidx.compose.ui.input.key.KeyEvent
 import androidx.compose.ui.text.input.EditCommand
 import androidx.compose.ui.text.input.ImeAction
 import androidx.compose.ui.text.input.ImeOptions
 import androidx.compose.ui.text.input.PlatformTextInputService
 import androidx.compose.ui.text.input.TextFieldValue
-import org.w3c.dom.events.KeyboardEvent
 
 internal interface InputAwareInputService {
     fun getOffset(rect: Rect): Offset
-    fun processKeyboardEvent(keyboardEvent: KeyboardEvent)
+    fun processKeyboardEvent(keyboardEvent: KeyEvent): Boolean
 }
 
 internal abstract class WebTextInputService : PlatformTextInputService, InputAwareInputService {
 
-    private var backingTextArea: BackingTextArea? = null
+    private var mBackingDomInput: BackingDomInput? = null
```

Comment:
:) `mBackingDomInput` - a naming convention from java?

I don't mind, just never seen it in Kotlin, or at least in Compose sources.

Reply:
@eymar it was slip of a finger, pushed more typical name )